cypress/integration/pages/mediaAssetPage/tests.js

```diff
@@ -54,6 +55,19 @@ export const testsThatFollowSmokeTestConfig = ({ service, pageType }) => {
         },
       );
     });
+
+    it('should render a media player', () => {
+      cy.request(`${config[service].pageTypes[pageType].path}.json`).then(
+        ({ body }) => {
+          const { assetUri } = body.metadata.locators;
+          const { versionId } = body.content.blocks[0].versions[0];
+          const { language } = body.metadata;
+          cy.get(
+            `[src*="${envConfig.liveRadioIframeBaseUrl}/ws/av-embeds/cps${assetUri}/${versionId}/${language}"]`,
```

Comment:
What kinda element is this?

Reply:
iframe on canonical, amp-iframe on amp

Reply:
``iframe[src="${envConfig.liveRadioIframeBaseUrl}/ws/av-embeds/media/${serviceId}/${id}/${lang}"]`,`

Reply:
This node can be an iframe or an amp-iframe depending on format

Reply:
Can you be explicit in that please? like the live radio pages

Reply:
👍 split this test into the amp and canonical sections